.github/workflows/binder-nbval.yml

```
@@ -0,0 +1,25 @@
+name: NBVAL on binder container
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  repo2docker-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build Docker Container with repo2docker
+      uses: jupyterhub/repo2docker-action@master
+      with:
+        NO_PUSH: 'true'
+        IMAGE_NAME: "python4compscience-binder"
```

```suggestion
        IMAGE_NAME: "python4compscience"
```

Naming it binder is useful here to explicitly state that the base image is the repo2docker one, but in the make file `DOCKER_RUN` assumes that the image name is `python4compscience`, so having a different name here breaks that.

Since this doesn't push the image and the image is only built for the tests, giving it the slightly more ambiguous `python4compscience` name is probably fine.